string library_path1
string library_path2
---
string result
